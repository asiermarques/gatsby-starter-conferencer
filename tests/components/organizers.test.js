import React from "react"
import renderer from "react-test-renderer"

import Organizers from "../../src/components/organizers"
import OrganizersConfigFixture from "../__fixtures/OrganizersConfig"

describe("Organizers", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Organizers organizers={OrganizersConfigFixture}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})