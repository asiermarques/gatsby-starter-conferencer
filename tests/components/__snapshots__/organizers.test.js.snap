// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organizers renders correctly 1`] = `
Array [
  <h3>
    Organizers
  </h3>,
  <div
    class="row justify-content-start"
  >
    <div
      class="col-6 col-md-3"
    >
      <a
        href="https://asiermarques.com"
        rel="noopener noreferrer"
        target="_blank"
        title="Sara Smith"
      >
        <img
          alt="Sara Smith"
          class="img-thumbnail"
          src="https://randomuser.me/api/portraits/women/40.jpg"
        />
      </a>
    </div>
    <div
      class="col-6 col-md-3"
    >
      <a
        href="https://asiermarques.com"
        rel="noopener noreferrer"
        target="_blank"
        title="Sara Smith"
      >
        <img
          alt="Sara Smith"
          class="img-thumbnail"
          src="https://randomuser.me/api/portraits/women/40.jpg"
        />
      </a>
    </div>
    <div
      class="col-6 col-md-3"
    >
      <a
        href="https://asiermarques.com"
        rel="noopener noreferrer"
        target="_blank"
        title="Sara Smith"
      >
        <img
          alt="Sara Smith"
          class="img-thumbnail"
          src="https://randomuser.me/api/portraits/women/40.jpg"
        />
      </a>
    </div>
  </div>,
]
`;
